### PR TITLE
Lt 1086 mobile login activity

### PR DESCRIPTION
**Pull request by Waqas-Zamir**
* Added missing validations for settings
* Exposed endpoint to publish login activity only for mobile clients
*POST* `/api/accounts/{accountId}/activities/login`

### CODE REVIEW

**Comment by dafigerdjikova** [comment]
Shouldn't we updates Axle Client as well?

File: src/Axle/Authorization/MobileClientAndAccountOwnerHandler.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        protected override Task HandleRequirementAsync(AuthorizationHandlerContext context, MobileClientAndAccountOwnerRequirement requirement)
+        {
+            var routeData = this.contextAccessor.HttpContext.GetRouteData();
+            var accountId = routeData.Values["accountId"]?.ToString();
+
+            var isMobileClient = context.User.HasClaim(JwtClaimTypes.ClientId, "nova_mobile");
+
+            var isAccountOwner = context.User.HasClaim(AuthorizationClaims.Accounts, accountId);
+
+            if (isMobileClient && isAccountOwner)
```
Comment (Waqas-Zamir) [inline src/Axle/Authorization/MobileClientAndAccountOwnerHandler.cs]: @dafigerdjikova @iliyan-georgiev-ph I have added a policy handler to make sure if user is owner of the account as well as request is coming form nova_mobile client

**Comment by Waqas-Zamir** [comment]
Updating client is not required because this endpoint is specific for mobile clients and they won't use this client. And we don't need to consume this in any other place

**Comment by dafigerdjikova** [comment]
> Updating client is not required because this endpoint is specific for mobile clients and they won't use this client. And we don't need to consume this in any other place

how are we going to call the endpoint from Donut, or you plan mobile app to call Axle

**Pull request merged**